### DISCUSSION TITLE
Tablet bug

### DISCUSSION THREAD

**Discussion by 3k0r0** [ppy/osu#26684] [Q&A] [answered]
### Type

Table bug

### Bug description

Whenever you try to play lazer on a chromebook, if you reach an end of the play area, the cursor goes flying into a random corner.  Not just for gameplay but in the menu too

### Screenshots or videos

https://github.com/ppy/osu/assets/135353366/15943706-8ca0-4ac2-9212-4c0c4a671973

### Version

2024.114.0

### Logs

Game crashed :/

**Comment by bdach** [comment]
Do you have "high precision mouse" enabled in settings? If yes, can you try disabling and seeing if it helps?

**Reply by 3k0r0** [reply]
I did actually try this and it just made the tablet not work at all 

Vid:
Uploading 20240123_160641_1.mp4…

**Reply by bdach** [reply]
Not sure then sorry. Chromebooks aren't exactly on our list of hardware that we guarantee full support for...

**Reply by 3k0r0** [reply, marked as answer]
I would like to add tho that this does also effect Linux gameplay too but it stops and goes back to normal if you change over to window mode (when on chromebook)
